backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -14,5 +16,23 @@ public class AvailabilityService : BaseService, IAvailabilityService
         public AvailabilityService(EasyMeetsCoreContext context, IMapper mapper) : base(context, mapper)
         {
         }
+
+        public async Task CreateAvailabilitySlot(NewAvailabilitySlotDto slotDto)
+        {
+            var entity = _mapper.Map<AvailabilitySlot>(slotDto);
+            
+            await _context.AvailabilitySlots.AddAsync(entity);
+
+            await _context.SaveChangesAsync();
```

Comment:
Rewrite to use one saveChanges(). And also link entities by navigation property instead of is (if possible)